frontend/packages/dev-console/integration-tests/features/topology/topology-smoke_test.feature

```diff
@@ -0,0 +1,165 @@
+   When user clicks on Zoom In option
+   And user sees the chart area is zoomed
+   And user clicks on Zoom Out option
+   And user sees the chart area is zoomed out
+   And user clicks on Fit to Screen option
+   And user sees the nodes fitting within chart area
+   And user clicks on Reset View option
+   Then user sees the chart area is reset to original
+
+@regression, @smoke
+Scenario: Verify Application grouping
+   Given User is in topology with an application cluster
+   When User clicks on an applicaton grouping
+   And User can see application sidebar
+   And user can confirm the workload information present under resources in the sidebar
+   And user can view "Add to Application" and "Delete Application" in the Action menu
+   And user right click on Application to open context menu
+   Then user can view action menu options
+
+@regression
+Scenario: Display options in topology
+   Given User is in topology with workloads
+   When User clicks on Display Options on top of topology
+   And User sees "Pod Count", "Event Sources" if present and "Labels" under "Show" and "Expand" have options according to their presence like "Application Groupings", "Helm Releases", "Knative Services" and "Operator Groupings"
+   And user select "Pod Count" which is deselected by default
+   And user checks the workloads which shows pod count instead of buider images
+   And user deselect "Application Groupings" in the Expand section
+   Then user can see workload squashed in Application grouping
+
+@regression, @smoke
+Scenario: Checking List view in topology
+   Given user is in topology with workloads 
+   When user clicks on List view button
+   And user verifies the "Group by" filter on top
+   Then user check nodes are present under applications
+
+@regression, @smoke
+Scenario: Deleting a workload
+   Given user is in the topology with different workloads
+   When user right clicks on the node 
+   And user checks delete option in the menu
+   And user clicks on workload to open sidebar
+   And user clicks on Action menu
+   And user clicks delete workload(deployment, deployment-config etc.)
+   And user sees "Delete" modal box to open
+   And user checks "Delete dependent objects of this resource" to be checked
+   And user clicks on "Delete"
+   Then user sees the workload disappearing from topology
+
+@regression, @smoke
+Scenario: Editing a workload
+   Given user is in the topology with different workloads
+   When user right clicks on the node 
+   And user sees context menu
+   And user clicks on "Edit App name"(like Edit golang)/ "Edit Service" (In case of knative)
+   And user can see Edit form
+   And user verifies that name of the application and route option is not editable
+   And user verifies that Application grouping, git url builder image version and advanced option can be edited
+   And user clicks on save
+   Then user can see the changes
+
+@regression, @smoke
+Scenario: Add revision to knative service through sidebar
```

Comment:
Remove this scenario from this file as we will be having this in different file and we can add it there. For now, we can remove this. We should keep this file for the general topology features. @makambalaji  WDYT about this

Reply:
This is present as a part of smoke test feature in the test cases sheet

Reply:
We are going to cover it in the Knative part, I don't think it's gonna makes sense to add it here as well. It's just duplication of scenario that will be present in the Knative.